lib/core/utils/collect-results-from-frames.js

```diff
@@ -27,7 +27,12 @@ function collectResultsFromFrames(
   frames.forEach(frame => {
     const tabindex = parseInt(frame.node.getAttribute('tabindex'), 10);
     const focusable = isNaN(tabindex) || tabindex >= 0;
+
     const rect = frame.node.getBoundingClientRect();
+    let width = parseInt(frame.node.getAttribute('width'), 10);
+    let height = parseInt(frame.node.getAttribute('height'), 10);
+    width = isNaN(width) ? rect.width : width;
+    height = isNaN(height) ? rect.height : height;
```

Comment:
This prevents margin CSS from affecting the width/height (otherwise in Chrome a width and height of 1 would result in a rect size of 5x5).